project-base/app/web/imageResizer.php

```diff
@@ -66,11 +78,39 @@
 
 function getImageFromUrl(string $url): void
 {
-    $image = file_get_contents($url);
+    $ch = curl_init($url);
+    curl_setopt_array($ch, [
+        CURLOPT_RETURNTRANSFER => true,
+        CURLOPT_FOLLOWLOCATION => true,
+        CURLOPT_TIMEOUT => 5,
+        CURLOPT_CONNECTTIMEOUT => 2,
+        CURLOPT_USERAGENT => 'ImageProxy/1.0',
```

Comment:
Do we want to fine-tune the curl a bit more? 

Auntie GPT suggested these options, and it somehow makes sense to me. Please validate if you wish.

```
CURLOPT_HTTPHEADER => [
    'Accept: image/*',
    'Cache-Control: no-cache',
],

// Prevents MITM attacks when accessing HTTPS URLs. I don't know if we access the backend on https
CURLOPT_SSL_VERIFYPEER => true,

// Prevents infinite redirect loops.
CURLOPT_MAXREDIRS => 5,

// Makes curl_exec() return false on HTTP errors (like 404 or 500), instead of returning the error page content. 
CURLOPT_FAILONERROR => true,

// Enables automatic handling of gzip/deflate encoded responses. 🤷‍♂️ I have no idea
CURLOPT_ENCODING => '',